scripts/run_benchmarking

```diff
@@ -2,38 +2,75 @@
-# NB: takes ~1h30 to complete
-/install/bin/pixels-performance-jace 6 /data/BBBC/NIRHTa-001.ome.tiff /data/out/bbbc-jace.ome.tiff /data/results/bbbc-pixeldata-linux-jace.tsv
+mkdir -p "${resultpath}"
+mkdir -p "${outpath}"
+rm "${resultpath}"/* || echo "No results"
+rm "${outpath}"/* || echo "No output"
+
+for test in bbbc mitocheck tubhiswt; do
+    input=unknown
+    case "$test" in
+        bbbc)
+            input=${datapath}/BBBC/NIRHTa-001.ome.tiff
+        ;;
+        mitocheck)
+            input=${datapath}/mitocheck/00001_01.ome.tiff
+        ;;
+        tubhiswt)
+            input=${datapath}/tubhiswt-4D/tubhiswt_C0_TP0.ome.tif
+        ;;
+    esac
+
+    # Java tests
+    (
+        mvn -P metadata -Dtest.iterations=${iterations} -Dtest.input="$input" -Dtest.output=${outpath}/${test}-java.ome.xml -Dtest.results=${resultpath}/${test}-metadata-linux-java.tsv exec:java
+        for i in $(seq ${iterations}); do
+            mvn -P metadata -Dtest.iterations=1 -Dtest.input="$input" -Dtest.output=${outpath}/${test}-java-${i}.ome.xml -Dtest.results=${resultpath}/${test}-metadata-linux-java-${i}.tsv exec:java
+        done
+
+        mvn -P pixels -Dtest.iterations=${iterations} -Dtest.input="$input" -Dtest.output=${outpath}/${test}-java.ome.tiff -Dtest.results=${resultpath}/${test}-pixeldata-linux-java.tsv exec:java
+        for i in $(seq ${iterations}); do
+            mvn -P pixels -Dtest.iterations=1 -Dtest.input="$input" -Dtest.output=${outpath}/${test}-java-${i}.ome.tiff -Dtest.results=${resultpath}/${test}-pixeldata-linux-java-${i}.tsv exec:java
+        done
+    )
+
+    # C++ tests
+    (
+        ${binpath}/metadata-performance ${iterations} "$input" ${outpath}/${test}-cpp.ome.xml ${resultpath}/${test}-metadata-linux-cpp.tsv
+        for i in $(seq ${iterations}); do
+            ${binpath}/metadata-performance 1 "$input" ${outpath}/${test}-cpp-${1}.ome.xml ${resultpath}/${test}-metadata-linux-cpp-${1}.tsv
+        done
+
+        ${binpath}/pixels-performance ${iterations} "$input" ${outpath}/${test}-cpp.ome.tiff ${resultpath}/${test}-pixeldata-linux-cpp.tsv
+        for i in $(seq ${iterations}); do
+            ${binpath}/pixels-performance 1 "$input" ${outpath}/${test}-cpp-${i}.ome.tiff ${resultpath}/${test}-pixeldata-linux-cpp-${i}.tsv
+        done
+    )
+
+    # JACE tests
+    (
+        cd $BF_JACE_HOME
+        # Can throw boost: mutex exception
+        ${binpath}/metadata-performance-jace ${iterations} "$input" ${outpath}/${test}-jace.ome.xml ${resultpath}/${test}-metadata-linux-jace.tsv || true
+        for i in $(seq ${iterations}); do
+            ${binpath}/metadata-performance-jace 1 "$input" ${outpath}/${test}-jace-${i}.ome.xml ${resultpath}/${test}-metadata-linux-jace-${i}.tsv || true
+        done
+
+        jace_iterations=${iterations}
+        # NB: takes ~1h30 to complete
+        if [ "$test" == "BBBC" ]; then
```

Comment:
Should this be `bbbc` (lowercase) instead? At the moment, JACE is running 20 executions for the BBBC plate on beluga